src/config/dynamic-theme-fixes.config

```diff
@@ -17739,6 +17739,24 @@ INVERT
 
 ================================
 
+shipstation.com
+
+INVERT
+.note-icon-3F9ryyv
```

Comment:
Here's the notes when they are inverted by INVERT .note-icon-3F9ryyv
https://postimg.cc/mzhTtFDc

This makes it much easier to tell which notes have an actual note in them that needs to be checked.

This is the notes when they are not inverted:
https://postimg.cc/MXtTVt44

Reply:
Alright feel free to take part of my suggestions then.